project/AppDependencies.scala

```diff
@@ -6,24 +6,24 @@ object AppDependencies {
 
   val compile = Seq(
     ws,
-    "uk.gov.hmrc" %% "play-conditional-form-mapping" % "0.2.0",
-    "uk.gov.hmrc" %% "play-reactivemongo" % "6.7.0",
+    "uk.gov.hmrc" %% "play-conditional-form-mapping" % "1.0.0-play-26",
+    "uk.gov.hmrc" %% "simple-reactivemongo" % "7.19.0-play-26",
     "uk.gov.hmrc" %% "logback-json-logger" % "4.6.0",
-    "uk.gov.hmrc" %% "govuk-template" % "5.35.0-play-25",
-    "uk.gov.hmrc" %% "play-health" % "3.14.0-play-25",
-    "uk.gov.hmrc" %% "play-ui" % "7.39.0-play-25",
-    "uk.gov.hmrc" %% "http-caching-client" % "8.3.0",
-    "uk.gov.hmrc" %% "bootstrap-play-25" % "4.11.0",
-    "uk.gov.hmrc" %% "play-language" % "3.4.0",
```

Comment:
Just comment here, this library is our blocker. We can't use it with actual version of play.

I decided to remove it, we're not using this and moreover language implementation is easy. I'll implement it in our service or prepare PR with upgrade for this library. But now we don't need it